### PR TITLE
cses divisor sum?

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
closes #2893

File: solutions/cses-1082.mdx [hunk 1]
```diff
@@ -2,264 +2,163 @@
...
 }
-tcTU> T lstTrue(T lo, T hi, U f) {
-	lo --; assert(lo <= hi); // assuming f is decreasing
-	while (lo < hi) { // find first index such that f is true
-		T mid = lo+(hi-lo+1)/2;
-		f(mid) ? lo = mid : hi = mid-1;
+
+int main() {
+	long long n;
+	std::cin >> n;
```
Comment (jessechoe10) [inline solutions/cses-1082.mdx:77]: Cringe af why std::cin >> n;

**Comment by jessechoe10** [comment]
Add this Youtube tutorial as well: https://www.youtube.com/watch?v=JqWiWJQOQyU. It's pretty good 🤡

File: solutions/cses-1082.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/cses-1082.mdx:77]: cope and seethe

File: solutions/cses-1082.mdx [hunk 2]
```diff
@@ -2,264 +2,168 @@
...
+
+	long long total = 0;
+	long long at = 1;  //
```
Comment (ryanchou-dev) [inline solutions/cses-1082.mdx:85]: empty comment?

File: solutions/cses-1082.mdx [hunk 3]
```diff
@@ -2,264 +2,168 @@
...
 
+</JavaSection>
+<PySection>
+
+```py
+MOD = 10 ** 9 + 7
+
+
+def total_sum(start: int, end: int) -> int:
+	"""Returns the sum of all numbers in [start, end]."""
```
Comment (ryanchou-dev) [inline solutions/cses-1082.mdx:145]: why not just use `#`?

**Review by ryanchou-dev — COMMENTED** [review]
yayy

File: solutions/cses-1082.mdx (hunk 3 above)
Comment (SansPapyrus683) [reply solutions/cses-1082.mdx:145]: idk lol
if you want you can change it

Comment (ryanchou-dev) [reply solutions/cses-1082.mdx:145]: err, ok

doesn't rly matter anyway

**Review by ryanchou-dev — APPROVED** [review]

**Review by HarryWangATX — APPROVED** [review]

**Pull request merged**